client/clientservice/src/request_service.cpp

```diff
@@ -52,9 +52,6 @@ Status RequestServiceImpl::Send(ServerContext* context, const Request* proto_req
         case (concord_client_pool::InvalidArgument):
           status.set_value(grpc::Status(grpc::StatusCode::INVALID_ARGUMENT, "Invalid argument"));
           break;
-        case (concord_client_pool::TimedOut):
-          status.set_value(grpc::Status(grpc::StatusCode::DEADLINE_EXCEEDED, "Timeout"));
-          break;
```

Comment:
This needs to be left. This switch should work with OperationResult instead of SubmitResult.

Remove TimedOut and InvalidArgument from this structure:
enum SubmitResult {
  Acknowledged,  // The request has been queued for submission
  Overloaded,    // There is no available client to the moment to process the
  // request
  InvalidArgument,
  TimedOut,
  ClientUnavailable,  // There are clients in the queue but none of them are connected to enough replicas
};